backend/multi_api_orchestrator.py

```
@@ -0,0 +1,750 @@
+        """Extract numeric stats from scraper output"""
+        try:
+            lines = output.split('\n')
+            for line in lines:
+                if all(keyword.lower() in line.lower() for keyword in keywords):
+                    # Extract number from line
+                    import re
+                    numbers = re.findall(r'\d+', line)
+                    if numbers:
+                        return int(numbers[0])
+            return 0
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)